src/components/settings/settings-root-proxy/settings-root-proxy.test.js

```diff
@@ -0,0 +1,166 @@
+import { render } from '@testing-library/react';
+
+import Harness from '../../../../test/jest/helpers/harness';
+
+import SettingsRootProxy from './settings-root-proxy';
+
+const proxyTypes = {
+  errors: [],
+  isFailed: false,
+  isLoading: false,
+  items: [
+    {
+      attributes: {
+        id: 'EZProxy',
+        name: 'EZProxy',
+      },
+      id: 'EZProxy',
+    },
+  ],
+};
+
+const rootProxy = {
+  data: {
+    attributes: {
+      id: 'root-proxy-id',
+      proxyTypeId: 'ezproxy',
+    },
+    id: 'root-proxy-id',
+    type: 'rootProxyType',
+  },
+  errors: [],
+  isFailed: false,
+  isLoading: false,
+  isUpdated: false,
+};
+
+const componentToRender = props => (
+  <Harness>
+    <SettingsRootProxy
+      onSubmit={() => {}}
+      proxyTypes={proxyTypes}
+      rootProxy={rootProxy}
+      {...props}
+    />
+  </Harness>
+);
```

Comment:
Why you can not use it inside `renderSettingsRootProxy`? By the way, if you want to create a separate function, so please, it's naming should starts with a verb (for ex. `getComponent`, `getTestComponent` etc.)

Reply:
I can't use it inside `renderSettingsRootProxy` because I need to rerender the component in the tests, and `renderSettingsRootProxy` contains `render` function, which doesn't seem to make sense to put inside the `rerender`. So I moved the component itself to a separate function, so that it could be reused in `rerender` just by passing needed props.
Regarding naming, yes, my bad, I didn't notice that I forgot to fix it, sorry.